daemon/daemon.go

```diff
@@ -407,20 +407,12 @@ func (daemon *Daemon) reserveName(id, name string) (string, error) {
 
 		conflictingContainer, err := daemon.GetByName(name)
 		if err != nil {
-			if strings.Contains(err.Error(), "Could not find entity") {
-				return "", err
-			}
```

Comment:
Just wondering; could we make those containers show up in `docker ps -a` in stead? And allow users to remove them? I'm not a big fan of too much "magic" / "smart" behavior (especially when poorly documented, lol)